### ISSUE TITLE
Special Pets that you don't own should show only grey pawprint

### ISSUE THREAD

**Issue by Alys** [HabitRPG/habitica#10866]
When you do not own a Special Pet, the only indicator that you see in the Stable should be the grey pawprint, not the black pawprint or the greyed-out image of the pet.

For example, for the Royal Purple Gryphon, you should see only this light grey pawprint for the Pet when you don't own it:

![image](https://user-images.githubusercontent.com/1495809/48965954-acdcda80-f013-11e8-9931-2c195a436967.png)

Currently, you see the black pawprint if you don't own the pet but do own a Gryphon egg and Royal Purple potion (you shouldn't see the black pawprint because you can never hatch that pet). You see the greyed-out image of the pet if you own the mount but not the pet. I.e., both of these are wrong:

![image](https://user-images.githubusercontent.com/1495809/48965988-034a1900-f014-11e8-8645-106e50bf9228.png)

![image](https://user-images.githubusercontent.com/1495809/48965986-fdecce80-f013-11e8-991e-ab3ce6f7c70f.png)

In summary, the code that decides what kind of icon to show for a Pet should have different handling for Special Pets so that it shows only the grey pawprint regardless of what other items the user owns.

----

_NB we do have https://github.com/HabitRPG/habitica/issues/9498 to completely hide the pawprints for **some** pets and mounts but even after that issue has been fixed, this new one would still be a problem for the pets that should show pawprints (e.g., if you buy a special mount with an Hourglass without buying the pet)._

**Comment by Scanace** [comment]
Hello, I want to help, can I take care of this issue?

**Comment by Scanace** [comment]
Hello @Alys , I already fixed this on my local installation of the website, I'll wait ulterior instructions from you!
Meanwhile I'll commit on my local repository.

**Comment by Alys** [comment]
@Scanace oh sweet! Thanks! Please do a PR for it. :)

**Comment by SabreCat** [comment]
Had some conversation with @saraolson about this at random this week--should we be showing pawprints at all, for special pets? It's not actionable information, and can be very sadness-inducing to see a slot for a pet it isn't and will never be possible to get (as in my case, where I have a lone gray pixel paw for Cerberus, reminding me that I missed the Kickstarter...).

**Comment by SabreCat** [comment]
Ah, that's #9498. Never mind, already recorded!
